sectionedrecyclerview/src/main/java/com/cruxlab/sectionedrecyclerview/lib/BaseSectionAdapter.java

```diff
@@ -266,6 +266,18 @@ public final void notifyItemMoved(int fromPos, int toPos) {
         }
     }
 
+    /**
+     * Notifies SectionDataManager that items in section were replaced.
+     * <p>
+     * Note that this method does not notify about changes in header, you can use
+     * {@link SectionManager#updateSection(int)} for complete section update.
```

Comment:
Please add a link to a similar `RecyclerView` method.
I'd also mention an alternative approach of using `notifyDataSetChanged` combined with `notifyHeaderChanged` in the situation above.

Reply:
Done, thx for review!